searchform.php

```diff
@@ -7,16 +7,24 @@
 
 // Exit if accessed directly.
 defined( 'ABSPATH' ) || exit;
+
+$uid = wp_unique_id( 's-' ); // The search form specific unique ID for the input.
+
+$aria_label = '';
+if ( isset( $args['aria_label'] ) && ! empty( $args['aria_label'] ) ) {
+	$aria_label = 'aria-label="' . esc_attr( $args['aria_label'] ) . '"';
+}
 ?>
 
-<form method="get" id="searchform" action="<?php echo esc_url( home_url( '/' ) ); ?>" role="search">
-	<label class="sr-only" for="s"><?php esc_html_e( 'Search', 'understrap' ); ?></label>
-	<div class="input-group">
-		<input class="field form-control" id="s" name="s" type="text"
-			placeholder="<?php esc_attr_e( 'Search &hellip;', 'understrap' ); ?>" value="<?php the_search_query(); ?>">
-		<span class="input-group-append">
-			<input class="submit btn btn-primary" id="searchsubmit" name="submit" type="submit"
-			value="<?php esc_attr_e( 'Search', 'understrap' ); ?>">
-		</span>
+<form class="search-form" method="get" action="<?php echo esc_url( home_url( '/' ) ); ?>" <?php echo $aria_label; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped -- Escaped above. ?>>
+	<div role="search">
+		<label class="sr-only" for="<?php echo $uid; ?>"><?php echo esc_html_x( 'Search for:', 'label', 'understrap' ); ?></label>
```

Comment:
Should we escape `$uid` here and below?

Reply:
`wp_unique_id()` is considered save as it only adds an integer after the prefix and in this case our prefix is safe too.

```
function wp_unique_id( $prefix = '' ) {
    static $id_counter = 0;
    return $prefix . (string) ++$id_counter;
}
```
Code taken from  https://developer.wordpress.org/reference/functions/wp_unique_id/